### DISCUSSION TITLE
Rich Text Editor Text Align isActive

### DISCUSSION THREAD

**Discussion by SeDemal** [mantinedev/mantine#5030] [Feature requests]
### What package has an issue

@mantine/tiptap

### Describe the bug

Most controls in the Rich Text Editor have the isActive parameter when it matters but not the text aligns for some reason
[Rich Text Editor Controls.tsx line 134 to 156](https://github.com/mantinedev/mantine/blob/3ea73f8efd0f9daf0f2b65bc52d00a2e779270ed/src/mantine-tiptap/src/RichTextEditorControl/controls.tsx#L134)
You can programmatically go around that by setting the background parameter but I think it'd be better to make it behave the same as the other controls from the start.

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

6.0.0 but link shows 7.1.2

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by rtivital** [comment]
isActive is not added to these controls on purpose – in case it is added, text-align: left control will be always active. Additionally, it will be hard to implmenet for rtl direction where default text-align is right.

**Reply by SeDemal** [reply]
Ah that make sense, would there be a possibility to have the "active" field like in the custom control available then?
I noticed that modifying the bg field actually conflicts with the hovering bg change.

**Reply by rtivital** [reply]
You should be able to override both background and hover background colors with `className` prop

**Reply by SeDemal** [reply]
![image](https://github.com/mantinedev/mantine/assets/26098587/938239f5-b570-4e88-8789-facabaffdd51)
I've seen that the color of the icon also changes. Looking at other buttons I've seen that the "data-active" tag was the one responsible for handling that change. It is a bit superfluous to have to handle all of it if I could just set that tag to true. I've manually done it in the browser and it does just work. Any way to do that programatically?
